woocommerce/woocommerce-gutenberg-products-block
# Shopper → Cart → Can proceed to checkout

Fixes #5763

### Notes

This PR adds an e2e test to verify that a shopper can proceed to checkout from the Cart block by clicking on the `Proceed to Checkout` button.

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/cart-proceed-to-checkout.test.js` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.
